Concepts/Elevator/Services/ElevatorService.cs

```diff
@@ -0,0 +1,129 @@
+﻿using ElevatorSystem.Entities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace ElevatorSystem.Services
+{
+
+    public class ElevatorService
```

Comment:
create the interface for elevator service.

Reply:
done